h/migrations/versions/b32d09564cf5_correct_timestamps_of_elife_annotations.py

```diff
@@ -0,0 +1,6394 @@
+"""
+Correct the timestamps of imported eLife annotations.
+
+Set the timestamps of eLife annotations that were imported using an API script
+to their correct timestamps according to the data file that eLife gave us.
+
+Revision ID: b32d09564cf5
+Revises: 9bcc39244e82
+Create Date: 2017-11-23 15:56:54.292312
+"""
+
+from __future__ import unicode_literals
+
+from datetime import datetime
+import logging
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql as pg
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.ext.mutable import MutableDict
+from sqlalchemy.orm import sessionmaker
+
+from h.db import types
+
+
+revision = 'b32d09564cf5'
+down_revision = '9bcc39244e82'
+
+
+log = logging.getLogger(__name__)
+
+
+Base = declarative_base()
+Session = sessionmaker()
+
+
+# The string format of the timestamps below.
+FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+
+# The groupid of the public group, in the production DB, that the annotations
+# to be modified belong to.
+GROUPID = "ijDGibQE"
+
+
+# The list of annotations we're going to modify (identified by their
+# imported_id values in the DB) and the timestamp values we're going to set for
+# them.
+TIMESTAMPS = [
```

Comment:
I think this is fine to have hard-coded in the migration (as I think we discussed before), but would it be possible to have it defined *after* the migration code? That would mean not having to scroll through a few thousand lines of dictionary to see the code itself.

Reply:
Yeah, I can just move it, agree that's nicer